### PR TITLE
[DOCS-9760] updates to resource catalog

### PR DESCRIPTION
**Pull request by aliciascott**
### What does this PR do? What is the motivation?
Updating out-dated screenshots and slight workflow changes in the UI

### Merge instructions

Merge readiness:
- [x] Ready for merge

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/infrastructure/resource_catalog/_index.md [hunk 1]
```diff
@@ -82,27 +82,27 @@ The Security tab provides a security-centric view of your infrastructure and all
...
 
-To access the relevant cloud console for any resource in your list, use the **Open Cloud Console** button to be redirected.
+{{< img src="/infrastructure/resource_catalog/resource-catalog_security_tab_list.png" alt="Resource Catalog Security tab sorting by the list view" >}}
 
-{{< img src="/infrastructure/resource_catalog/resource-catalog-doc-img-4.png" alt="Resources owned by the cloud-networks team grouped by resource type" width="100%">}}
+**Pro Tip**: If you use [Datadog Teams][4], select the **Teams** toggle on the left panel, then select the toggle for the teams to which you're assigned to view only resources assigned to those teams. In addition, export your Resource Catalog list as a CSV file from the top right corner of the list.
```
Comment (rtrieu) [inline content/en/infrastructure/resource_catalog/_index.md:95]:
```suggestion
**Note**: If you use [Datadog Teams][4], select the **Teams** toggle on the left panel, then select the toggle for the teams to which you're assigned to view only resources assigned to those teams. In addition, export your Resource Catalog list as a CSV file from the top right corner of the list.
```

Comment (rtrieu) [reply content/en/infrastructure/resource_catalog/_index.md:95]: I don't believe we do "Pro Tips"

Comment (aliciascott) [reply content/en/infrastructure/resource_catalog/_index.md:95]: good catch! I think that was there before but I didn't catch that :)

**Review by rtrieu — APPROVED** [review]

**Pull request merged**